src/pl-comp.c

```diff
@@ -4786,7 +4786,7 @@ argKey(Code PC, int skip, word *key)
 	goto again;
 #endif
       default:
-	Sdprintf("Unexpected VM code %d at %p\n", c, PC);
+	Sdprintf("Unexpected VM code %ld at %p\n", c, PC);
```

Comment:
codes (c) are uintptr_t.   As noted, `l` doesn't work.   As `code` in decoded form is also small, best use %d and cast to `int`.   Should there be cases in encoded form, then cast to `size_t` and use `%zd`.